views/admin-options.php

```diff
@@ -0,0 +1,188 @@
+                        </p>
+                    </td>
+                </tr>
+                <tr>
+                    <th scope="row"><label for="yoti_sdk_id">Yoti Client SDK ID</label></th>
+                    <td>
+                        <input
+                            name="yoti_sdk_id"
+                            type="text"
+                            id="yoti_sdk_id"
+                            value="<?php esc_attr_e($data['yoti_sdk_id']); ?>"
+                            class="regular-text code"
+                        />
+                        <p>
+                            <code>Yoti Client SDK ID</code> identifies your Yoti Hub application.
+                            This value can be found in the Hub, within your application section, 
+                            in the keys tab.
+                        </p>
+                    </td>
+                </tr>
+                <tr>
+                    <th scope="row"><label for="yoti_company_name">Company Name</label></th>
+                    <td>
+                        <input
+                            name="yoti_company_name"
+                            type="text"
+                            id="yoti_company_name"
+                            value="<?php esc_attr_e($data['yoti_company_name']); ?>"
+                            class="regular-text code"
+                        />
+                        <p>
+                            <code>Company Name</code> to replace WordPress wording in the 
+                            warning message on the login form.
+                        </p>
+                    </td>
+                </tr>
+                <tr>
+                    <th scope="row"><label for="yoti_pem">Yoti PEM File</label></th>
+                    <td>
+                        <?php if (!empty($data['yoti_pem']['name'])) { ?>
+                            <div class="pem-file">
+                                <code>
+                                    <strong>Current file:</strong> <?php esc_html_e($data['yoti_pem']['name']); ?>
+                                </code>
+                                <label>
+                                    <input 
+                                        type="checkbox"
+                                        name="yoti_delete_pem"
+                                        value="1"
+                                        <?php checked(!empty($data['yoti_delete_pem'])); ?>
+                                    /> Delete this PEM file
+                                </label>
+                            </div>
+                        <?php } ?>
+                        <input name="yoti_pem" type="file" id="yoti_pem" />
+                        <p>
+                            <code>Yoti PEM File</code> is the application pem file. 
+                            It can be downloaded only once from the Keys tab in your Yoti Hub.
```

Comment:
Since we can redownload now, shall we update this? 
```suggestion
                            It can be downloaded from the Keys tab in your Yoti Hub.
```